### PR TITLE
fix(frontend): to run in advanced mode

### LINKED ISSUE #1238: Feishu - cannot be triggered by advanced mode
## Describe the bug
Cannot be triggered by advanced mode

## To Reproduce
1. run feishu through advanced mode
2. See error: Advanced Pipeline: Unsupported Data Provider Plugin Detected!

## Expected behavior
Can ben triggered by advanced mode

### PR DESCRIPTION
**Pull request by warren830**
# Summary

Added feishu to advanced mode

### Key Points

- [ ] This is a breaking change
- [ ] New or existing documentation is updated

### Description
Describe what this PR does, and aims to solve in a few sentences.

### Does this close any open issues?
Closes #1238

### Current Behavior
Can not run through advanced mode

### New Behavior
We can load feishu in advanced mode and run

### Screenshots
image
### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: config-ui/src/data/Providers.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ const Providers = {
   JIRA: 'jira',
   GITHUB: 'github',
   REFDIFF: 'refdiff',
-  GITEXTRACTOR: 'gitextractor'
+  GITEXTRACTOR: 'gitextractor',
+  FEISHU: 'feishu'
```
Comment (e2corporation) [inline config-ui/src/data/Providers.js:18]: Further down this file on `Ln 122` there is a `ProviderIcons` constant that also needs a new entry for Feishu.

1. Download official Feishu logo SVG and store somewhere under `@/images`
2. Import SVG as `FeishuProviderIcon`
3. Add Icon entry Object to `ProviderIcons` following existing pattern

**Comment by e2corporation** [comment]
@warren830 `config-ui/src/components/pipelines/StageTaskName.jsx` also Needs updating to show the task label for **Feishu**. Please perform a functional test on Config-UI Pipelines by running a Pipeline with Feishu config after making these changes.

```
# Add on Ln 72 after JENKINS
{task.plugin === Providers.FEISHU && (<>{ProviderLabels.FEISHU}</>)}
```

**Comment by warren830** [comment]
> StageTaskName
How about this? I am so bad at dealing with arts
image

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**